QUARKUS-5858.md

```diff
@@ -0,0 +1,30 @@
+# QUARKUS-5858 Tech preview Support for OAuth2 Demonstrating Proof of Possession
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-5858
+
+Upstream PR: https://github.com/quarkusio/quarkus/pull/45891
+
+Upstream documentation:
+* https://quarkus.io/version/main/guides/security-oidc-bearer-token-authentication#demonstrating-proof-of-possession-dpop
+
+## Preview status
+DPoP is currently considered tech-preview in quarkus.
+
+## Scope of testing
+Verify quarkus works with authentication method "Demonstrating Proof of Possession" (DPoP)
+A new test class will be implemented in `security/keycloak-oidc-client-extended`.
+Tests will verify authentication using DPoP, which will cover test cases:
+* Happy path - correctly accessing resource using DPoP authentication.
+* Authentication failures:
+  * No DPoP header.
```

Comment:
Good, but just having unauthorized doesn't feel enough, let's also try sending valid token using `Authorization` scheme while DPoP is required, so that we know that it is always enforced (judging by the code it is...)

Reply:
I've added this case.